package-lock.json

```diff
@@ -38,6 +38,7 @@
         "merge-stream": "2.0.0",
         "pkg-dir": "4.2.0",
         "prismjs": ">=1.24.0",
+        "spec-up": "^0.10.6",
```

Comment:
@darrellodonnell this is what i referred to in the slack message

Reply:
removed